benchmark_runner/benchmark_operator/workload_flavors/perf_ci/hammerdb/hammerdb_data_template.yaml

```diff
@@ -18,20 +18,26 @@ shared_data:
   metrics_profile: node-metrics.yml
   #metrics_profile: https://raw.githubusercontent.com/cloud-bulldozer/benchmark-operator/master/roles/kube-burner/files/metrics.yaml
 mariadb:
+  mariadb_limits_cpu: 16
+  mariadb_limits_memory_GB: 16
   db_image: centos8-mariadb103-container-disk:latest
   db_pass: mysql
   db_port: 3306
   db_type: mariadb
   db_user: root
   db_server_pod: mariadb-deployment.mariadb-db
 mssql:
+  mssql_limits_cpu: 32
+  mssql_limits_memory_GB: 16
   db_image: centos8-mssql2019-container-disk:latest
   db_pass: s3curePasswordString
   db_port: 1433
   db_type: mssql
   db_user: SA
   db_server_pod: mssql-deployment.mssql-db
 postgres:
+  postgres_limits_cpu: 16
+  postgres_limits_memory_GB: 16
```

Comment:
Am I understanding correctly that there was no limit prior?  That would result in an implicit limit for Kata of 2 GB RAM and give a single core to the VM.

Reply:
There was a prior limitation it was 
limits_cpu: 16
memory_GB: 16